src/scss/includes/font.scss

```diff
@@ -31,12 +31,15 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-.icon-chevron-left:before {
+.icon-chevron-left2:before {
```

Comment:
This breaks the poi panel chevron that still uses `icon-chevron-left` and `icon-chevron-right`

Reply:
corrigé